pkg/apis/core/v1beta1/constants/types_constants.go

```diff
@@ -781,6 +781,8 @@ const (
 	DefaultVPNRange = "192.168.123.0/24"
 	// DefaultVPNRangeV6 is the default IPv6 network range for the VPN between seed and shoot cluster.
 	DefaultVPNRangeV6 = "fd8f:6d53:b97a:1::/96"
+	// ReservedSeedServiceRange is the IPv4 network range for the "kubernetes" service used by apiserver-proxy
+	ReservedSeedServiceRange = "240.0.0.0/8"
```

Comment:
I am a bit unhappy with the naming. This range will only show up in shoot clusters, but its name suggests it is connected to the seed service range. Maybe something like `ReservedSeedServiceReplacementRange` fits better, but I open to other names.

Reply:
`ReservedKubeApiServerMappingRange`?